src/players/player1326_states.py

```diff
@@ -0,0 +1,107 @@
+from abc import abstractmethod
+from bet import Bet
+from players.player import Player
+
+
+class Player1326State:
+    def __init__(self, player: Player) -> None:
```

Comment:
The type hint for player should be `Player1326`. `Player1326State` should not store any other player than `Player1326`. In case client wrongly passes other class, mypy will be able to report if proper type hinting is used. Also subclasses of `Player1326State` access attribute `player.outcome`. Only `Player1326` has outcome attribute.